docs/explain.md

```diff
@@ -0,0 +1,46 @@
+# Explain
+
+When reviewing inferred axioms, such as after applying the [`reason`](/reason) command,
+you may like more information about why a particular axiom was inferred:
+
+    robot explain -i uvula.ofn -r ELK --axiom "'uvular muscle' SubClassOf 'muscle organ' and 'part of' some 'jaw region'" -o uvular_muscle.md
+
+Here we provide an axiom in Manchester syntax. If that relationship is entailed by the 
+axioms asserted in [uvula.ofn](/examples/uvula.ofn), we can use the selected reasoner (here, ELK)
+to compute an _explanation_—a minimal set of axioms required to derive that inference. Reviewing 
+inference explanations is an extremely useful tool for debugging ontology classification issues.  
+The explanation is printed in Markdown format, which can be easily pasted and rendered within 
+GitHub issues:
+
+    ## [uvular muscle](http://purl.obolibrary.org/obo/UBERON_0010235) SubClassOf [muscle organ](http://purl.obolibrary.org/obo/UBERON_0001630) and ([part of](http://purl.obolibrary.org/obo/BFO_0000050) some [jaw region](http://purl.obolibrary.org/obo/UBERON_0011595)) ##
```

Comment:
I don't know if we need both the markdown code and the rendered markdown... I think the rendered markdown would be sufficient for the docs. @jamesaoverton - what do you think?

Reply:
I removed the plaintext version.